.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "deps/SDL2-CS"]
 	path = deps/SDL2-CS
-	url = https://github.com/goatcorp/SDL2-CS
```

Comment:
We should leave these on goatcorp, they share a tree anyway if you forked

Reply:
done